java/test/jmri/jmrix/loconet/messageinterp/LocoNetMessageInterpretTest.java

```diff
@@ -6809,6 +6817,20 @@ public void testExpendedAccyWrites() {
         Assert.assertEquals("write 4: Ext Accy Addr 1 CV 12 to 0", ""
                 + "Extended Accessory Decoder CV Write: Address 1 CV 12 with 0.\n",
                 LocoNetMessageInterpret.interpretMessage(l, "LT", "LS", "LR"));
+
+        // [ED 0B 7F 51 15 3C 67 6D 03 5E 49] Extended Accessory Decoder CV 'Write': Address 500 CV 260, write 222.
+        l = new LocoNetMessage(new int[] {0xED, 0x0B, 0x7F, 0x51,
+            0x15, 0x3C, 0x67, 0x6D, 0x03, 0x5E, 0x49});
+        Assert.assertEquals("write 5: Ext Accy Addr 500 CV 260 to 222", ""
+                + "Extended Accessory Decoder CV Write: Address 500 CV 260 with 222.\n",
+                LocoNetMessageInterpret.interpretMessage(l, "LT", "LS", "LR"));
+        
```

Comment:
It would be better if the code would attempt to check a few different CV numbers, _especially_ CV numbers that exercise the CV numbers that were changed.  At a minimum, the tests should exercise CV Numbers 256, 257, 512, 513, and 1024, to verify bits 8 and 9, and the maximum CV number  value.